core/iwasm/aot/aot_runtime.c

```diff
@@ -9,6 +9,7 @@
 #include "../common/wasm_runtime_common.h"
 #include "../common/wasm_memory.h"
 #include "../interpreter/wasm_runtime.h"
+#include "platform_common.h"
```

Comment:
Please include `bh_platform.h` instead, in the original design, bh_platform.h includes all necessary header files for `core/shared/utils` and `core/shared/platform`, and other component should include it and don't include "platform_xxx.h".